### PR TITLE
Update jersey client

### PR DESCRIPTION
**Pull request by amontenegro**

### CODE REVIEW

**Comment by amontenegro** [comment]
[Upgrade Jersey client to 3.0.5](https://trello.com/c/oZ31uUGX/8196-upgrade-jersey-client-to-305)

File: orcid-core/src/main/java/org/orcid/core/manager/v3/read_only/impl/EmailManagerReadOnlyImpl.java [hunk 1]
```diff
@@ -122,18 +118,14 @@ public boolean isPrimaryEmailVerified(String orcid) {
...
+            LOGGER.error(message);            
             throw nre;
         } catch (javax.persistence.NonUniqueResultException nure) {
             String alternativePrimaryEmail = emailDao.findNewestPrimaryEmail(orcid);
             emailDao.updatePrimary(orcid, alternativePrimaryEmail);
             
             String message = String.format("User with orcid %s have more than one primary email, so, we are setting the latest modified primary as the primary one", orcid);
-            LOGGER.error(message);
-            
-            slackManager.sendSystemAlert(message);
```
Comment (Camelia-Orcid) [inline orcid-core/src/main/java/org/orcid/core/manager/v3/read_only/impl/EmailManagerReadOnlyImpl.java:136]: the notification in slack is no longer needed?

File: orcid-core/src/main/java/org/orcid/core/manager/v3/read_only/impl/EmailManagerReadOnlyImpl.java [hunk 2]
```diff
@@ -122,18 +118,14 @@ public boolean isPrimaryEmailVerified(String orcid) {
             emailDao.updatePrimary(orcid, alternativePrimaryEmail);
             
             String message = String.format("User with orcid %s have no primary email, so, we are setting the newest verified email, or, the newest email in case non is verified as the primary one", orcid);
-            LOGGER.error(message);
-            
-            slackManager.sendSystemAlert(message);
```
Comment (Camelia-Orcid) [inline orcid-core/src/main/java/org/orcid/core/manager/v3/read_only/impl/EmailManagerReadOnlyImpl.java:127]: The notification in slack is no longer needed?

File: orcid-core/src/main/java/org/orcid/core/security/OrcidUserDetailsServiceImpl.java [hunk 3]
```diff
@@ -159,8 +155,7 @@ private String retrievePrimaryEmail(ProfileEntity profile) {
             
             String message = String.format("User with orcid %s have no primary email, so, we are setting the newest verified email, or, the newest email in case non is verified as the primary one", orcid);
             LOGGER.error(message);
-            
-            slackManager.sendSystemAlert(message);
```
Comment (Camelia-Orcid) [inline orcid-core/src/main/java/org/orcid/core/security/OrcidUserDetailsServiceImpl.java:163]: The notification in slack is no longer needed?

File: orcid-core/src/main/java/org/orcid/core/security/OrcidUserDetailsServiceImpl.java [hunk 4]
```diff
@@ -169,7 +164,6 @@ private String retrievePrimaryEmail(ProfileEntity profile) {
             String message = String.format("User with orcid %s have more than one primary email, so, we are setting the latest modified primary as the primary one", orcid);
             LOGGER.error(message);
             
-            slackManager.sendSystemAlert(message);
```
Comment (Camelia-Orcid) [inline orcid-core/src/main/java/org/orcid/core/security/OrcidUserDetailsServiceImpl.java:172]: The notification in slack is no longer needed?

**Pull request merged**